CHANGELOG.md

```diff
@@ -136,6 +136,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### API Breaking Changes
 
+* (baseapp) [#16415](https://github.com/cosmos/cosmos-sdk/issues/16415) `IsAllowed` from the `CircuitBreaker` interface now takes a `context.Context` instead of a `sdk.Context` and returns `bool, error`.
```

Comment:
So, this just got backported to v0.46 and v0.47, if you make a PR updating the interface targeting to release branch (and fast before we release them :D); then we do not need the changelog as there will not be any breaking changes.

Reply:
ok, lemme try

Reply:
I am off today as well, but basically we only need to do the changes in baseapp interface and backport that, as circuit is a go.mod so it isn't backported in previous version.

Reply:
Opened https://github.com/cosmos/cosmos-sdk/pull/16430